krauter1-craps/src/controller/CrapsController.java

```diff
@@ -298,34 +298,34 @@ public void keyReleased(KeyEvent e) {
 
 
     }
-
-    private void updateWallet(int theValue) {
-
-        if (counter1 == 0) {
-            myInitialTempValue = myCash;
-        }
-        if (gameLogic.getRoundUpdate()) {
-            counter1 = 0;
-        } else {
-            counter1++;
-        }
-        System.out.println("Initial bet " + myInitialTempValue);
-        if (gameLogic.getWinValue() == false) {
-            myCash = myCash - theValue;
-            System.out.println("updated wallet " + myCash);
-            myTempValue = myInitialTempValue - myCash;
-            System.out.println("Amount on the table " + myTempValue);
-            System.out.print(" = " + myInitialTempValue);
-            System.out.print(" - " + myCash);
-            setWallet(myCash);
-        } else {
-            System.out.println("Temp value " + myTempValue);
-            System.out.println("Cash " + myCash);
-            setWallet(myTempValue*2 + myCash);
-            gameLogic.setMyWinValue(false);
-
-        }
-    }
+//
+//    private void updateWallet(int theValue) {
```

Comment:
Why did you do this here?

Reply:
I am removing code that I intend to rewrite after writing new pseudo code.